app/models/mixins/scanning_operations_mixin.rb

```diff
@@ -29,15 +29,15 @@ def save_metadata_op(xmlFile, type, jobid = nil)
     true
   end
 
-  def agent_job_state_op(jobid, state, message = nil)
+  def agent_job_message_op(jobid, message)
     _log.info "jobid: [#{jobid}] starting"
     begin
       Timeout.timeout(WS_TIMEOUT) do
         MiqQueue.put(
           :class_name  => "Job",
-          :method_name => "agent_state_update_queue",
-          :args        => [jobid, state, message],
-          :task_id     => "agent_job_state_#{Time.now.to_i}",
+          :method_name => "agent_message_update_queue",
+          :args        => [jobid, message],
```

Comment:
With the following method signature, how will this work?
```ruby
def self.agent_state_update_queue(jobid, _state, message = nil)
```

Any way of wrapping a spec around this?